templates/prometheus.yaml.erb

```diff
@@ -17,4 +17,4 @@ if @prometheus_v2
     full_config['remote_read'] = remote_read_configs
 end
  -%>
-<%= full_config.to_yaml(options = {:line_width => -1}).gsub(/source_labels: ".+?"/) { |x| x.gsub('"', '') } -%>
+<%= full_config.to_yaml(options = {:line_width => -1}).gsub(/.*:\W"\[.*\]"/) { |x| x.gsub('"', '') } -%>
```

Comment:
@benpollardcts thank you for reporting and providing a PR. Have you seen #108 ? Is this also an issue for you?